examples/map.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/AlecAivazis/survey"
+)
+
+// the questions to ask
+var simpleQs = []*survey.Question{
+	{
+		Name: "name",
+		Prompt: &survey.Input{
+			Message: "What is your name?",
+		},
+		Validate: survey.Required,
+	},
+	{
+		Name: "color",
+		Prompt: &survey.Select{
+			Message: "Choose a color:",
+			Options: []string{"red", "blue", "green"},
+		},
+		Validate: survey.Required,
+	},
+}
+
+func main() {
```

Comment:
Sorry to ask this, but i'm confused what the intended use case is for doing this sort of arbitrary type conversion from a string? Seems like if the user wants a specific type out, they should just use the prompt that collects that type, or do the conversion from the provided type to the intended one themselves.